t/obs/openSUSE:Jump:15.2:Images:ToTest/print_openqa.before

```diff
@@ -0,0 +1,54 @@
+/usr/share/openqa/script/client isos post --host localhost \
```

Comment:
`/usr/bin/openqa-cli` should be used instead of the deprecated client. However, I think this is out-of-scope for this PR.

Reply:
Yes, absolutely. All I actually did was add the new `.xml` file and let the scripts do the rest.